packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -164,15 +127,30 @@ export default class {
 
   onNetworkTypeChange(usePrivateNetwork) {
     if (!usePrivateNetwork) this.model.subnet = null;
+    this.getOrderData();
+  }
+
+  // bug in oui-numeric: on-change event is fired before model is updated
+  onNodeNumberChange() {
+    this.$timeout(() => this.getOrderData());
+  }
```

Comment:
Not an issue i think, it's expected behavior.
If you would the right value, you have to pass from HTML template this `modelValue`.

Reply:
Oh ok, I changed the comment to explain why we use a $timeout here. https://github.com/ovh/manager/pull/6905/commits/15ac9b6b17cc909010729c53281b391ff78b7e59